README.md

```diff
@@ -57,7 +57,7 @@ def duplicates?(array)
 end
 ```
 
-Snippet 5 - Big O:
+Snippet 5 - Big O: O(n^2) because each loop is O(n) (and you multiply them together when nested)
```

Comment:
@sudocrystal Apologies for the late reply! I believe this is O(n*m) because they're two separate variables that could potentially have different sized arrays.